tachyon/crypto/sumcheck/sumcheck_prover_msg.h

```diff
@@ -0,0 +1,57 @@
+// Copyright 2022 arkworks contributors
+// Use of this source code is governed by a MIT/Apache-2.0 style license that
+// can be found in the LICENSE-MIT.arkworks and the LICENCE-APACHE.arkworks
+// file.
+
+#ifndef TACHYON_CRYPTO_SUMCHECK_SUMCHECK_PROVER_MSG_H_
+#define TACHYON_CRYPTO_SUMCHECK_SUMCHECK_PROVER_MSG_H_
+
+#include <utility>
+#include <vector>
+
+#include "tachyon/base/buffer/copyable.h"
+
+namespace tachyon {
+namespace crypto {
+
+template <typename F>
+struct SumcheckProverMsg {
+  // Evaluations on P(0), P(1), P(2), ...
+  std::vector<F> evaluations;
```

Comment:
Is it more proper to change `math::UnivariateEvluations<F, MaxDegree>`?

Reply:
Sounds good!